### ISSUE TITLE
b-form-group feedback text should be inserted (rather than toggling display)

### ISSUE THREAD

**Issue by tinynow** [bootstrap-vue/bootstrap-vue#6307]
### Describe the bug
Best practices for assistive technology suggest inserting contents of a live region that is not hidden via `display: none;`, or at least putting the `aria-live` or alert role on a parent and toggling the visibility of children.
[https://www.davidmacd.com/blog/test-aria-live-display-none.html](https://www.davidmacd.com/blog/test-aria-live-display-none.html)

### Steps to reproduce the bug

- Go to https://bootstrap-vue.org/docs/components/form-group
- Trigger the feedback messages in the demo component
- Inspect the rendered markup

#### Libraries:

   - BootstrapVue: 2.21.2

### Additional context
#6300 Addresses a related issue with redundant aria. It seems like removing the `role: alert` and going with the aria-live solution is the recommendation: https://www.w3.org/WAI/tutorials/forms/notifications/#inline
As with that issue, I am not basing this on testing, only research of the specs and the testing article above.
